pepper-apis/src/main/java/org/broadinstitute/ddp/constants/ConfigFile.java

```diff
@@ -105,6 +105,7 @@
     public static final String PUBSUB_TASKS_SUB = "pubsub.pubSubTasksSubscription";
     public static final String PUBSUB_TASKS_RESULT_TOPIC = "pubsub.pubSubTasksResultTopic";
     public static final String PUBSUB_TASKS_SUBSCRIBER_AWAIT_RUNNING_TIMEOUT = "pubsub.pubSubTaskSubscriberAwaitRunningTimeout";
+    public static final String PUBSUB_EVENT_ACTION_TOPIC = "pubsub.pubSubEventActionTopic";
```

Comment:
Why do we need another topic? Can we just use the current pubsub topic that we use to communicate with DSM?

Reply:
The existing PubSubTask service works a different way: PubSubTasks service used to listen tasks from DSM for DSS. So it is task to be executed by DSS. There are 2 special topics used for it:
- "pubsub.pubSubTasksSubscription" - the scubscription listening by DSS for tasks from DSM;
- "pubsub.pubSubTasksResultTopic" - topic where DSS publishes result of a task execution.
Therefore we can not use this service in this task (where we have an opposite purpose: to send a task from DSS to DSM). So I suggest to use a separate topic for this.

Reply:
Can you connect with @Irakli-Khaladze or whoever is working on the DSM-side for this feature? And make sure we're on the same page and agree on the topics and such?